### PR TITLE
Change note styling on max word count

### PR DESCRIPTION
**Pull request by henriettemoe**

### CODE REVIEW

File: src/semantics.ts [hunk 1]
```diff
@@ -253,6 +253,12 @@ export const semantics: Readonly<[H5PFieldGroup, H5PBehaviour, H5PL10n]> = [
         default: "Saved",
         type: H5PFieldType.Text,
       },
+      {
+        label: "Word limit exceeded label",
+        name: "dialogNoteLimitExceeded",
+        default: "Limit exceeded",
```
Comment (boyum) [inline src/semantics.ts:259]: Perhaps something less generic?

```suggestion
        default: "Too many words",
```

File: src/utils/semantics.utils.ts [hunk 2]
```diff
@@ -16,6 +16,15 @@ export const itemDialog: Array<H5PField> = [
         type: H5PFieldType.Boolean,
         default: true,
       },
+      {
+        label: "Maximum words",
```
Comment (boyum) [inline src/utils/semantics.utils.ts:20]: Only a suggestion

```suggestion
        label: "Maximum number of words",
```

**Review by boyum — APPROVED** [review]
Looks good! 🦄

File: src/semantics.ts (hunk 1 above)
Comment (henriettemoe) [reply src/semantics.ts:259]: Yes, I agree ✨

File: src/utils/semantics.utils.ts (hunk 2 above)
Comment (henriettemoe) [reply src/utils/semantics.utils.ts:20]: Yes, will add this

**Pull request merged**